modules/misc/gtk.nix

```diff
@@ -10,23 +10,13 @@ let
 
   toGtk3Ini = generators.toINI {
     mkKeyValue = key: value:
-      let
-        value' = if isBool value then
-          (if value then "true" else "false")
-        else
-          toString value;
-      in "${key}=${value'}";
+      let value' = if isBool value then boolToString value else toString value;
+      in "${escape [ "=" ] key}=${value'}";
   };
 
   formatGtk2Option = n: v:
-    let
-      v' = if isBool v then
-        (if v then "true" else "false")
-      else if isString v then
-        ''"${v}"''
-      else
-        toString v;
-    in "${n} = ${v'}";
+    let v' = if isBool v then boolToString value else toString v;
```

Comment:
Hum,
The value is no longer wrapped in double quotes if a string?
Could this introduce any bugs?

Reply:
you are right,  I overlooked this. String values for gtk2 settings must be quoted in the form `key = "value"` but antiquotation will remove one pair of quotes so it needs to be wrapped.